flaky/_flaky_plugin.py

```diff
@@ -233,7 +233,11 @@ def _add_flaky_report(self, stream):
             `file`
         """
         stream.write('===Flaky Test Report===\n\n')
-        stream.write(self._stream.getvalue())
+        value = self._stream.getvalue()
+        try:
+            stream.write(value)
+        except UnicodeEncodeError:
+            stream.write(value.encode('utf-8', 'replace'))
```

Comment:
Why not ``stream.write(ensure_unicode_string(self._stream.getvalue()))``? This seems to duplicate the behavior.